.zuul.yaml

```diff
@@ -1,4 +1,31 @@
 ---
+- job:
+    name: test-cfg-cookiecutter
```

Comment:
Please stick to the convention of starting the job name with the repo name. Also `cfg-cookiecutter-test` is a bit generic, maybe `-tox` at least?